census/keras/trainer/model.py

```diff
@@ -69,13 +67,13 @@ def model_fn(input_dim,
     input_dim = units
 
   # Add a dense final layer with sigmoid function
-  model.add(layers.Dense(labels_dim, activation=sigmoid))
+  model.add(layers.Dense(labels_dim, activation=softmax))
   compile_model(model, learning_rate)
   return model
 
 def compile_model(model, learning_rate):
   model.compile(loss='categorical_crossentropy',
-                optimizer=keras.optimizers.SGD(lr=learning_rate),
+                optimizer=keras.optimizers.RMSprop(lr=learning_rate),
```

Comment:
Is RMSProp giving better result in this case?

Reply:
Yes, SGD gets stuck predicting the first class (having a ~75% base rate in the training dataset).
RMSprop (and Adam as far as I have tested), takes off towards .8 accuracy rather fast, with the default setup in this example.